dist/index.js

```diff
@@ -6691,7 +6691,22 @@ exports.getBinaryExtension = exports.mapArch = exports.mapPlatform = exports.get
 const os = __importStar(__nccwpck_require__(2037));
 const releases_json_1 = __importDefault(__nccwpck_require__(2387));
 // Get release info of a certain verion of ORAS CLI
-function getReleaseInfo(version) {
+function getReleaseInfo(version, url, checksum) {
+    // if customized ORAS CLI link and checksum are provided, version is ignored
+    if (url && checksum) {
+        return {
```

Comment:
Better place the comment in the corresponding code path.
```suggestion
    if (url && checksum) {
        // if customized ORAS CLI link and checksum are provided, version is ignored
        return {
```